service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ResourceExceptionHandler.java

```diff
@@ -7,47 +7,20 @@
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
-import uk.gov.hmcts.reform.fpl.exceptions.NoAssociatedUsersException;
-import uk.gov.hmcts.reform.fpl.exceptions.UnknownLocalAuthorityCodeException;
-import uk.gov.hmcts.reform.fpl.exceptions.UnknownLocalAuthorityDomainException;
+import uk.gov.hmcts.reform.fpl.exceptions.AboutToStartOrSubmitCallbackException;
 
 @ControllerAdvice
 public class ResourceExceptionHandler {
     private static final Logger logger = LoggerFactory.getLogger(ResourceExceptionHandler.class);
 
-    @ExceptionHandler(value = UnknownLocalAuthorityDomainException.class)
-    public ResponseEntity<Object> handleUnknownLocalAuthorityDomain(Exception exception) {
+    @ExceptionHandler(value = AboutToStartOrSubmitCallbackException.class)
+    public ResponseEntity<Object> handleAboutToStartOrSubmitCallbackException(AboutToStartOrSubmitCallbackException
+                                                                                  exception) {
```

Comment:
Please keep in one line, it just looks stupid.

Also change` ResponseEntity<Object>` to `AboutToStartOrSubmitCallbackResponse`.